playbooks/roles/rpc_maas/defaults/main.yml

```diff
@@ -63,17 +179,38 @@ maas_monitoring_zones:
   - mzlon
   - mzhkg
 
-# Set the fqdn extension if required, sample only
+#
+# maas_fqdn_extension: Sets the fully-qualified domain name (FQDN) extension that will be appended
+#                      to the short names for hosts that are specified in rpc_user_config. This
+#                      avoids the need to specify extremely long container names.
+#
+#   For example, if this variable was not set, a device called 12345-node1.mytestcluster.com would
+#   have to be named in openstack_user_config.yml as 12345-node1.mytestcluster.com for MaaS checks
+#   to be created correctly. However, if maas_fqdn_extension is set as .mytestcluster.com, the
+#   device can be named 12345-node1 in openstack_user_config.yml. This enables shorter container
+#   names while still enabling MaaS to be created properly.
+#
 # maas_fqdn_extension: .example.com
 maas_fqdn_extension:
 
-# Set the following to skip creating alarms for this device for disk utilisation monitoring
+#
+# maas_excluded_devices: Specifies devices that will be skipped when creating alarms for disk
+#                        utilisation monitoring.
+#
 # maas_excluded_devices: ['xvde']
 
-# Set the threshold for filesystem monitoring when you are not specifying specific filesystems.
+#
+# maas_filesystem_threshold: Sets the threshold for filesystem monitoring when you are not
+#                            specifying specific filesystems.
```

Comment:
This could be more specific. I would presume it is percent utilization but it would be good to be explicit.